generate.py

```diff
@@ -91,10 +101,44 @@ def generate_message(media, target, conf, result):
         raise RuntimeError('Invalid media {} for {}'.format(media, target))
 
 
+def count_tweets_in_twitter_url(url):
```

Comment:
All this browser faking and text processing would not be needed if we'd use the API... I'm still very much in favor of it. I'm generating the page every minute (or it should be every minute). 60 queries per hour are reasonable and should be well within the API rate limits.